File: common/src/main/java/guepardoapps/lucahome/common/services/IChangeService.java
package guepardoapps.lucahome.common.services;

import guepardoapps.lucahome.common.classes.Change;

public interface IChangeService extends ILucaService<Change> {
    String ChangeDownloadFinishedBroadcast = "guepardoapps.lucahome.common.services.change.download.finished";

    String ChangeDownloadFinishedBundle = "BirthdayDownloadFinishedBundle";
}

File: common/src/main/java/guepardoapps/lucahome/common/services/change/IChangeService.kt
package guepardoapps.lucahome.common.services.change

import guepardoapps.lucahome.common.models.change.Change
import guepardoapps.lucahome.common.services.common.ILucaService

interface IChangeService : ILucaService<Change> {
    fun get(type: String): Change?
}